config/initializers/delayed_job_config.rb

```diff
@@ -6,7 +6,7 @@
 Delayed::Worker.default_queue_name = :default
 
 if Rails.env.test? || Rails.env.development? # save a little time in testing/dev
-  Delayed::Worker.sleep_delay = 10
+  Delayed::Worker.sleep_delay = 5
```

Comment:
For rusty or unacquainted maintainers:

```suggestion
  Delayed::Worker.sleep_delay = 5 # minutes
```

Reply:
It's actually seconds, so I'll add that for clarity.